proto/osmosis/lockup/tx.proto

```diff
@@ -71,3 +72,15 @@ message MsgExtendLockup {
 }
 
 message MsgExtendLockupResponse { bool success = 1; }
+
+message MsgForceUnlock {
+  string owner = 1 [ (gogoproto.moretags) = "yaml:\"owner\"" ];
```

Comment:
Is this an SDK address? If so, shouldn't we add `(cosmos_proto.scalar) = "cosmos.AddressString"`? Maybe our proto tooling doesn't support this yet?

Reply:
Also, `  option (cosmos.msg.v1.signer) = "from_address";`

Reply:
what does `(cosmos_proto.scalar) = "cosmos.AddressString"` and `option (cosmos.msg.v1.signer) = "from_address";` do?

Orignial intention was to recieve it as string and then convert it to address